loose.json

```diff
@@ -42,22 +40,17 @@
         "space-infix-ops": 1,
         "space-unary-ops": 1,
         "wrap-iife": [2, "inside"],
-        "react/display-name": 1,
-        "react/jsx-boolean-value": 1,
-        "react/jsx-quotes": 1,
+        "react/jsx-boolean-value": [1, "never"],
+        "react/jsx-quotes": [1, "double", "avoid-escape"],
         "react/jsx-no-undef": 1,
-        "react/jsx-sort-props": 1,
-        "react/jsx-sort-prop-types": 1,
         "react/jsx-uses-react": 1,
         "react/jsx-uses-vars": 1,
         "react/no-did-mount-set-state": 1,
         "react/no-did-update-set-state": 1,
         "react/no-multi-comp": 1,
         "react/no-unknown-property": 1,
-        "react/prop-types": 1,
         "react/react-in-jsx-scope": 1,
         "react/self-closing-comp": 1,
-        "react/sort-comp": 1,
-        "react/wrap-multilines": 1
+        "react/wrap-multilines": 2
```

Comment:
As I recall the error msg when you don't do this is cryptic.  So could be helpful to error on this explicitly?  We'll see